From cba5dc6e08b03d3b174082ae8b1f1b12a9e4d96a Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Wed, 6 May 2015 15:08:22 -0400
Subject: [PATCH 322/323] Ticket 48158 - Remove cleanAllRUV task limit of 4

Bug Description:  There is a limit of 4 concurrent tasks, and this is too
                  low of a limit.

Fix Description:  There still needs to be a limit because each task creates
                  a new thread.  Setting limit to 64.

https://fedorahosted.org/389/ticket/48158

Reviewed by: rmeggins(Thanks!)

(cherry picked from commit cd8614a75852f4d05dc76d104867542f18384416)
(cherry picked from commit b97803f9797138a0d218ea2f2c6891fe798e2fcf)
---
 ldap/servers/plugins/replication/repl5.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/replication/repl5.h b/ldap/servers/plugins/replication/repl5.h
index 10c0869..6cec248 100644
--- a/ldap/servers/plugins/replication/repl5.h
+++ b/ldap/servers/plugins/replication/repl5.h
@@ -649,7 +649,7 @@ void set_cleaned_rid(ReplicaId rid);
 void cleanruv_log(Slapi_Task *task, int rid, char *task_type, char *fmt, ...);
 char * replica_cleanallruv_get_local_maxcsn(ReplicaId rid, char *base_dn);
 
-#define CLEANRIDSIZ 4 /* maximum number for concurrent CLEANALLRUV tasks */
+#define CLEANRIDSIZ 64 /* maximum number for concurrent CLEANALLRUV tasks */
 
 typedef struct _cleanruv_data
 {
-- 
1.9.3

